src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -395,17 +395,26 @@ export { SingleActiveFIMap };
  * @param {any} ownProps - the props
  */
 const mapStateToProps = (state: Partial<Store>, ownProps: any) => {
-  // pass in the plan id to get plan the get the jurisdicytion_id from the plan
+  // pass in the plan id to get plan the get the jurisdiction_id from the plan
   const plan = getPlanById(state, ownProps.match.params.id);
   let goals = null;
   let jurisdiction = null;
   let currentGoal;
   let pointFeatureCollection = defaultFeatureCollection;
   let polygonFeatureCollection = defaultFeatureCollection;
   let structures = null;
+  let plansByFocusArea: Plan[] = [];
   if (plan) {
     jurisdiction = getJurisdictionById(state, plan.jurisdiction_id);
     goals = getGoalsByPlanAndJurisdiction(state, plan.plan_id, plan.jurisdiction_id);
+    plansByFocusArea = getPlansArray(
+      state,
+      InterventionType.FI,
+      [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
+      null,
+      [],
+      plan.jurisdiction_name
```

Comment:
@p-netm What's the reasoning behind using `jurisdiction_name` vs `jurisdiction_id`?

Reply:
yeap, went with my initial insticts will modify that.